cmd/get/launch_plan.go

```diff
@@ -14,62 +14,62 @@ import (
 )
 
 const (
-	launchPlanShort = "Gets launch plan resources"
+	launchPlanShort = "Get launchplan resources"
```

Comment:
Can we have "launch plan" only? Because "launchplan" isn't the right word and it exists only in commands.

Reply:
Sure will do that